tasklist/hi.json

```
@@ -0,0 +1,520 @@
+{
+  " लेबल " : {
+    " APP_VENDOR " : " ऐप विक्रेता " ,
```

The translation label should be without leading or trailing spaces. I can't speak for the translation, please make sure that this is correct as well.

```suggestion
    "APP_VENDOR" : " ऐप विक्रेता " ,
```

This applies to all labels